packages/coinstac-ui/app/render/components/consortia/consortia-list.jsx

```diff
@@ -6,9 +6,11 @@ import { compose, graphql, withApollo } from 'react-apollo';
 import { ipcRenderer } from 'electron';
 import classNames from 'classnames';
 import { orderBy } from 'lodash';
-import {
-  Button, Menu, MenuItem, TextField, Typography,
-} from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
```

Comment:
whats up with all these import changes?

Reply:
When I tried to import elements using named imports, styles were broken.